ballerina-tests/tests/interceptors.bal

```diff
@@ -182,6 +316,16 @@ readonly service class InterceptMutation {
     }
 }
 
+readonly service class InterceptMutation2 {
+    *graphql:Interceptor;
+
+    isolated remote function execute(graphql:Context context, graphql:Field 'field) returns anydata|error {
+        var result = context.resolve('field);
```

Comment:
Do we need to call resolve here ?

Reply:
`context.resolve()` call makes sure that we are executing the actual resolver. So that it confirms we execute the actual resolver and then the rest of the interceptor logic. Also, there are test cases for interceptors that have not included the `resolve()` call.